kolibri/plugins/management/assets/src/vue/user-page/user-edit-modal.vue

```diff
@@ -1,19 +1,19 @@
 <template>
 
   <div class="user-edit-modal">
-    <modal v-ref:modal btntext="Edit">
+    <modal v-on:open="clear()" v-ref:modal btntext="Edit">
```

Comment:
wasn't sure if `@open` was a thing. I saw `@click` everywhere and assumed @ notation was only for HTML events